### ISSUE TITLE
NullPointerException when loading mb-module with EmailExt configured

### ISSUE THREAD

**Issue by moshezvi** [mjdetullio/multi-branch-project-plugin#69]
Jenkins 1.617, plugin: 0.2.2

Got this when restarting Jenkins:
```
Jun 28, 2015 7:17:07 AM hudson.model.AbstractProject createTransientActions
SEVERE: Could not load actions from hudson.plugins.emailext.EmailExtTemplateActionFactory@6bade3b3 for com.github.mjdetullio.jenkins
.plugins.multibranch.FreeStyleMultiBranchProject@71bfa007[alertsengine-mb]
java.lang.NullPointerException
        at com.github.mjdetullio.jenkins.plugins.multibranch.AbstractMultiBranchProject.getPublishersList(AbstractMultiBranchProject
.java:1232)
        at hudson.plugins.emailext.EmailExtTemplateActionFactory.createFor(EmailExtTemplateActionFactory.java:27)
        at hudson.model.AbstractProject.createTransientActions(AbstractProject.java:765)
        at com.github.mjdetullio.jenkins.plugins.multibranch.AbstractMultiBranchProject.createTransientActions(AbstractMultiBranchProject.java:1263)
        at hudson.model.AbstractProject.updateTransientActions(AbstractProject.java:754)
        at hudson.model.AbstractProject.onLoad(AbstractProject.java:339)
        at com.github.mjdetullio.jenkins.plugins.multibranch.AbstractMultiBranchProject.onLoad(AbstractMultiBranchProject.java:158)
        at hudson.model.Items.load(Items.java:322)
        at jenkins.model.Jenkins$17.run(Jenkins.java:2655)
        at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
        at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
        at jenkins.model.Jenkins$7.runTask(Jenkins.java:905)
        at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
        at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
```

**Comment by geoffroyjabouley** [comment]
I've been able to fix this issue, but i am not sure if it is a clean solution...

You will need:

* The latest version of email-ext plugin including this PR https://github.com/jenkinsci/email-ext-plugin/pull/106. Today, it is only available in the master branch, so you have to build yourself the plugin (git clone && mvn package)

* Modify the line https://github.com/mjdetullio/multi-branch-project-plugin/blob/master/src/main/java/com/github/mjdetullio/jenkins/plugins/multibranch/AbstractMultiBranchProject.java#L1238 and build the multi-branch plugin
```
return (templateProject != null) ? templateProject.getPublishersList() : null;
```

**Comment by mjdetullio** [comment]
I think this may have been fixed in 0.3
